packages/@lwc/engine-core/src/framework/attributes.ts

```diff
@@ -35,7 +34,7 @@ export const globalHTMLProperties: {
         error?: string;
         readOnly?: boolean;
     };
-} = assign(create(null), {
```

Comment:
Are there any implications if the prototype for `defaultDefHTMLPropertyNames` is no longer null?

From what I understand, `Object.getPrototypeOf(defaultDefHTMLPropertyNames)` would now return the prototype for an object literal but I'm not sure if it would cause any downstream issues.